backend/metadata_writer/src/metadata_writer.py

```diff
@@ -357,6 +357,20 @@ def get_output_template(obj):
                     outputs = get_output_template(obj)
                     pipeline_output_artifacts = {}
 
+                    # Add logging as output artifacts if enabled.
+                    archive_logs = os.environ.get('ENABLE_ARCHIVE_LOGS', 'false')
```

Comment:
FYI, I try to follow the same wording as argo's JSON config. Here I use underscore to split the words because KFP API are taking these flags via environment variables instead of a certain JSON syntax. 

https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/argo/workflow-controller-configmap.yaml#L27

If you still think we should change it to `ENABLE_LOGS_ARCHIVING`, then I can merge your suggestions.

Reply:
ARCHIVE_LOGS will work. But ENABLE_ARCHIVE_LOGS doesn't make sense.

If you are using ENABLE_ARTFICACT_TRACKING, this can follow the similar nomenclature.

Reply:
if we switch to Argo's wording, probably ENABLE_ARTIFACT_TRACKING should become TRACK_ARTIFACTS

Reply:
sure, let me remove all the `ENABLE` keywords

Reply:
done